api/service.go

```diff
@@ -67,8 +54,7 @@ func registerServices(ctx context.Context, grpcServer *grpc.Server, gwmux *runti
 	}
 
 	cfg := nodeObjReg[bootstrap.BootstrappedConfig].(config.Configuration)
-	jobsMan := nodeObjReg[jobs.BootstrappedService].(jobs.Manager)
-	return httpapi.Router(cfg, configService, invoiceUnpaidService, docService, transferService, jobsMan), nil
+	return httpapi.Router(nodeObjReg, cfg, configService), nil
```

Comment:
this call is not necessary to be done here, we can move it to `api/server.go` (like in my pr), this is mixup of new code and old code unnecessarily.